src/ripple/app/tx/Transaction.h

```diff
@@ -77,15 +76,20 @@ class Transaction
         boost::optional<std::uint64_t> const& ledgerSeq,
         boost::optional<std::string> const& status,
         Blob const& rawTxn,
-        Validate validate,
+        Application& app);
+
+    static
+    Transaction::pointer
+    transactionFromSQLValidated (
+        boost::optional<std::uint64_t> const& ledgerSeq,
```

Comment:
The `Transactions.LedgerSeq` column is defined as a `BIGINT UNSIGNED`, and is being read as a `std::uint64_t`, and I ain't touching that, at least not in this PR.